examples/README.md

```diff
@@ -13,4 +13,7 @@ The example consists of four agents:
     *"You are an english literature expert. Answer questions about english literature. Try to give context to your answers and provide quote from the books described. Your user is a native english speaker, with limited background in english literature."*
     3. **Translator Agent**: Handles requests and provides intelligent responses. Its system message says: 
     *"You are an expert translator from english to italian. Translate the provided text from english to italian. "*
-    4. **SequentialCoordinator**: Take care of coordinating the request handling from the user delagating the work to other agents in a predetermined order.
\ No newline at end of file
+    4. **SequentialCoordinator**: Take care of coordinating the request handling from the user delagating the work to other agents in a predetermined order.
+* [memory_agentic_chatbot_groq](./memory_agentic_chatbot_groq/README.md)
+
+    This guide walks you through running a practical example of a multi-agent system using Wise Agents. In this example, two agents (a web interface agent and an intelligent agent) are started, allowing you to experiment with agent communication and interaction in a simulated environment. This is the same descibed in [memory_agentic_chatbot](./memory_agentic_chatbot/), but instead of using a locla LLM it uses [https://groq.com/](https://groq.com/) for inference the model. Please obtain an API key from them and set env variable GROQ_API_KEY to start playing with this example
```

Comment:
s/same descibed/same as described
s/locla/local
s/for inference/to inference
s/from them/from https://console.groq.com/keys   (since I find it a bit hard to find)
s/set env variable GROQ_API_KEY/set the env variable `GROQ_API_KEY`
s/playing with this example/playing with this example. (full stop  needed at end)